### PR TITLE
Optimized package.json files

### PR DESCRIPTION
**Pull request by stevenvachon**
* Used repository shortcut syntax where possible
* Added repository directories for monorepo packages
* Alphabetically sorted npm scripts

### CODE REVIEW

File: packages/grafana-data/package.json [hunk 1]
```diff
@@ -9,17 +9,18 @@
   ],
   "repository": {
     "type": "git",
-    "url": "http://github.com/grafana/grafana.git"
+    "url": "http://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-data"
```
Comment (ryantxu) [inline packages/grafana-data/package.json]: what does this do?  what changes when you add directory?

**Review by ryantxu — APPROVED** [review]
looks fine -- I don't totally get why necessary, but seems good to be consistent :)

File: packages/grafana-data/package.json (hunk 1 above)
Comment (stevenvachon) [reply packages/grafana-data/package.json]: The link on npmjs.com will go to the package directory instead of the repo root

**Comment by stevenvachon** [comment]
Heheh, these were a few distracting annoyances.

**Pull request merged**